lib/utils/runOpen.js

```diff
@@ -3,11 +3,14 @@
 const open = require('opn');
 
 function runOpen(uri, options, log) {
-  let openOptions = {};
+  // https://github.com/webpack/webpack-dev-server/issues/1990
+  const openOptions = { wait: false };
   let openMessage = 'Unable to open browser';
 
   if (typeof options.open === 'string') {
-    openOptions = { app: options.open };
+    Object.assign(openOptions, {
+      app: options.open,
+    });
```

Comment:
```js
let openOptions = { wait: false };
// ...
if (typeof options.open === 'string') {
  openOptions = Object.assign({} openOptions, { app: options.open });
}
```

No need modify `openOptions` variable, it is very bad practice

Reply:
`openOptions = Object.assign({} openOptions, { app: options.open });` equals `Object.assign(openOptions, { app: options.open })`.

I don't think this is a very bad practice.

Reply:
Yes, but it is not good, we modify original object and it is bad practice because you can forget about this in future and it is lead to some problems when you refactor code. 

When you pass options to third party package(s) better always cloning object. Other package can modify object too so it can lead to very strange errors and situations.

Reply:
> When you pass options to third party package(s) better always cloning object. Other package can modify object too so it can lead to very strange errors and situations.

If you say, we should not assign to the same variable.

I don't have any strong opinions, so I'll change.